heudiconv/heuristics/bids_ME.py

```diff
@@ -21,9 +21,16 @@ def infotodict(seqinfo):
     subindex: sub index within group
     """
     bold = create_key('sub-{subject}/func/sub-{subject}_task-test_run-{item}_bold')
+    megre_mag = create_key('sub-{subject}/anat/sub-{subject}_part-mag_MEGRE')
+    megre_phase = create_key('sub-{subject}/anat/sub-{subject}_part-phase_MEGRE')
 
-    info = {bold: []}
+    info = {bold: [], megre_mag: [], megre_phase: []}
     for s in seqinfo:
         if '_ME_' in s.series_description:
             info[bold].append(s.series_id)
+        if 'GRE_QSM' in s.series_description:
+            if s.image_type[2] == 'M':
```

Comment:
note: this code block is not covered by tests... what data was supposed to have it covered?